backend/config/staffusers.json

```diff
@@ -0,0 +1,20 @@
+{
+  "readonly": [
+    "philip.dominguez@gsa.gov"
+  ],
+  "helpdesk": [
+    "alexander.steel@gsa.gov",
+    "robert.novak@gsa.gov",
+    "analyn.delossantos@gsa.gov",
+    "hassandeme.mamasambo@gsa.gov",
+    "marissa.henderson@gsa.gov",
+    "james.p.mason@gsa.gov",
+    "james.person@gsa.gov",
+    "leigh.cox@gsa.gov",
+    "rochelle.ribeiro@gsa.gov"
+  ],
+  "superuser": [
+    "matthew.jadud@gsa.gov",
```

Comment:
We may want to update this list as well but let's hear @danswick's thought.

Reply:
@danswick adding to that, Phil is the only person hanging out in `read-only` permission and that is because he is the only one without zendesk access. Should we change this?